examples/eks/iam.tf

```diff
@@ -0,0 +1,113 @@
+# IAM user required for CAST.AI
+
+provider "castai" {
+  api_url   = "https://console.cast.ai//"
```

Comment:
remove this url (our provider defaults to public api). By the way, api is located at https://api.cast.ai